report/report-generator.js

```diff
@@ -37,10 +37,13 @@ class ReportGenerator {
       .replace(/</g, '\\u003c') // replaces opening script tags
       .replace(/\u2028/g, '\\u2028') // replaces line separators ()
       .replace(/\u2029/g, '\\u2029'); // replaces paragraph separators
+    // terser does its own sanitization, but keep this basic replace for when
+    // we want to debug locally without minification.
```

Comment:
```suggestion
    // we want to generate a report without minification.
```
(not a huge distinction, maybe it doesn't matter)